tests/conftest.py

```diff
@@ -432,6 +432,18 @@ def uvm_plain_any(microvm_factory, guest_kernel, rootfs_ubuntu_22):
     return microvm_factory.build(guest_kernel, rootfs_ubuntu_22)
 
 
+guest_kernel_6_1_debug = pytest.fixture(
+    guest_kernel_fxt,
+    params=kernel_params("vmlinux-6.1*", artifact_dir=defs.ARTIFACT_DIR / "debug"),
+)
+
+
+@pytest.fixture
+def uvm_plain_debug(microvm_factory, guest_kernel_6_1_debug, rootfs_rw):
```

Comment:
we don't make use of the debug kernels in tests within this PR yet, do we?

Reply:
Yes, this fixture has no users at the moment, I just wanted to have an example on how to use it.